handlers/InitiateSalesforceSessionHandler.ts

```diff
@@ -100,6 +100,13 @@ export class InitiateSalesforceSession {
 					customDetail = undefined;
 				}
 
+				let prechatDetails: string | undefined;
+				if (this.data.room.customFields && this.data.room.customFields.prechatDetails) {
+					prechatDetails = this.data.room.customFields.prechatDetails;
+				} else {
+					prechatDetails = undefined;
+				}
+
```

Comment:
This can be shortened with 
```const prechatDetails: string = this.data.room.customFields?.prechatDetails;```

Can you please also do the same for other fields as well `buttonId`, `salesforceId`, `customDetail`?